libs/order/src/actions/order.actions.ts

```diff
@@ -11,10 +13,21 @@ export enum DaffOrderActionTypes {
   OrderListFailureAction = '[Order] Order List Failure Action'
 }
 
-export class DaffOrderLoad<T extends DaffOrder = DaffOrder> implements Action {
+/**
+ * Triggers the loading of the specified order.
+ *
+ * @param payload The order and guest cart ID.
+ */
+export class DaffOrderLoad<
+  T extends DaffOrder = DaffOrder,
+  V extends DaffCart = DaffCart
+> implements Action {
   readonly type = DaffOrderActionTypes.OrderLoadAction;
 
-  constructor(public payload: T['id']) {}
+  constructor(public payload: {
+    orderId: T['id'],
+    cartId?: V['id'],
+  }) {}
```

Comment:
what do you think about this sort of format vs:
```
constructor(public orderId: T['id'], public cartId?: V['id']) {}
```

Reply:
I don't have any strong preference either way. I guess yours has the benefit of not being a breaking change. Plus it exactly matches the driver call so its probably better that way.

Reply:
changed

Reply:
I was thinking that the usage looks a little cleaner: 
`new Action({ value1: value1, value2: value2})` vs `new Action(value1, value2)`